content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -762,7 +762,7 @@ prisma db pull
 ```code no-copy
 Introspecting based on datasource defined in schema.prisma …
 
-✔ Wrote Prisma data model into schema.prisma in 38ms
+✔ Wrote data model into schema.prisma in 38ms
```

Comment:
Code output, also needs a PR in prisma/prisma (search for "Wrote data model" there and make a PR for all occurrences)

Reply:
Scratch that, this CLI output is outdated already. There is no mention of "data model" in there any more.
Finding the correct output.

Reply:
```suggestion
✔ Introspected 2 models and wrote them into schema.prisma in 38ms
```